source/code/scxsystemlib/common/scxostypeinfo.cpp

```diff
@@ -52,7 +52,7 @@ using namespace SCXCoreLib;
 
 namespace {
 
-#if defined(linux)
+#if defined(linux) && defined(PF_DISTRO_SUSE)
```

Comment:
I get this with the current code on ubuntu15
/SCX2/bld-omsagent/pal/source/code/scxsystemlib/common/scxostypeinfo.cpp:65:13: error: ‘std::__cxx11::wstring {anonymous}::ExtractOSName(const wstring&)’ defined but not used [-Werror=unused-function]
     wstring ExtractOSName(const wstring& platformString)
             ^
cc1plus: all warnings being treated as errors
/SCX2/bld-omsagent/pal/build/Makefile.rules:13: recipe for target '/SCX2/bld-omsagent/pal/intermediate/Linux_ULINUX_1.0_x64_64_Debug/source/code/scxsystemlib/common/scxostypeinfo.o' failed
make: *** [/SCX2/bld-omsagent/pal/intermediate/Linux_ULINUX_1.0_x64_64_Debug/source/code/scxsystemlib/common/scxostypeinfo.o] Error 1